soletta/flow_node.go

```diff
@@ -38,6 +38,19 @@ func (fn *FlowNode) GetPort(name string, direction int) (portIndex uint16, ok bo
 	return t.GetPort(name, direction)
 }
 
+//Sets the data associated with the flow node
+func (fn *FlowNode) SetData(data interface{}) {
+	if data == nil {
+		return
```

Comment:
why ? ins't `nil` a valid option when one wants to remove the data associated with the flow node ?